code/game/objects/items/weapons/grenades/chem_grenade.dm

```diff
@@ -10,6 +10,8 @@
 	var/path = 0
 	var/obj/item/device/assembly_holder/detonator = null
 	var/list/beakers = new/list()
+	var/list/temp_reagents = new/list()
```

Comment:
Is there a reason to have vars like this on the object instead of in the code where they're being used? You're taking up memory with these lists.